target/linux/ramips/mt7620/base-files/lib/preinit/07_set_preinit_iface_ramips

```diff
@@ -0,0 +1,59 @@
+#!/bin/sh
+#
+# Copyright (C) 2013 OpenWrt.org
+#
+
```

Comment:
NAK on these scripts. I'll make a commit to remove default case in 02_network.

Reply:
Okay, that would be much better, thanks.